word_guess.rb

```diff
@@ -0,0 +1,182 @@
+#IN-CLASS - THURS FEB 25
+#game similar to hangman - cat loses z if wrong guess
+#game over shows cat leaving
+
+class WordGuess
+  attr_accessor :guessed_letters, :letter, :answer, :underscores, :num, :good,
```

Comment:
Most of the time, we want to use `attr_accessor`s for accessing these properties _outside_ of the `WordGuess` class. It seems like the code at the bottom that is running the game is not using these variables, but rather the methods. I wonder if you removed this attributes if everything would still run the way you expect?